daskhub-rhg/Chart.yaml

```diff
@@ -6,15 +6,15 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.1
+version: 0.1.0
```

Comment:
I'm kinda ambivalent on the need to bump this version and `appVersion` on line 15. It it's useful to you or users that's okay, but doing a chart up like this is kinda a hack anyways so I'd be fine if these versions didn't change. If we see a good reason to bump these versions, I'm up for the changing the practice.